stages/org.osbuild.chown

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util.path import in_tree
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["paths"],
+"properties": {
+  "paths": {
+    "type": "array",
+    "description": "Paths to operate on for changing ownership",
+    "items": {
+      "type": "string",
+      "pattern": "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$"
+    }
+  },
```

Comment:
I must admit that I prefer using a similar approach to what the `chmod` stage uses (https://github.com/osbuild/osbuild/blob/main/stages/org.osbuild.chmod#L23). In short, options are a dict, where the key is the actual path and value is another dict with the user / group to set on the path. The benefit of this approach is that it would allow us to use a single stage instance in a manifest to change ownership of various paths to different values. With the current approach, we would have to have one stage instance per the desired combination of user/group ownership.

Reply:
I may need to ask here for a bit more guidance :) I am not sure how I should continue

Reply:
I think that looking at the `chmod` stage is the key here. I'm not sure how to explain it in a different way than showing how it's done there 🤔 

```json
"options": {
  "additionalProperties": false,
  "properties": {
    "items": {
      "type": "object",
      "additionalProperties": false,
      "patternProperties": {
        "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$": {
          "type": "object",
          "required": [<required_properties>],
          "properties": {
            // properties that can be specified for a path, such as user, group, recursive, UID, GID, etc.
          }
        }
      }
    }
  }
}
```

Reply:
Of course feel free to reach out if this does not help 😉